src/TableContainerApp.vue

```diff
@@ -10,7 +10,11 @@
             :page="page"
             :selectable="selectable"
             :exactMatch="true"
-            :isExactMatchCaseSensitive="false"
+            :is-exact-match-case-sensitive="false"
+            :manage-table-properties="true"
+            :has-top-slot="true"
+            :fixed-table-head="true"
+            :pagination-with-page-input="true"
```

Comment:
You can replace `:prop-name="true"` with just `prop-name`. E.g. `<component has-top-slot />`

Reply:
But what's the difference if it's the same result? Yes we can omit prop value, but I think it would be better to specify the value explicitly.

Reply:
No difference. It would be less code to maintain. Again: We need code review for code style assessment and not for functional testing.

Reply:
We need code review for functional testing as well, hardly we need to merge not working code. The code styling, functional testing, optimization moments, discussions are important parts of the review.